src/System.Linq/tests/EnumerableTests.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public partial class EnumerableTests
+    {
+        [Fact]
+        public void Average()
+        {
+            var array = Enumerable.Range(1, 10).ToArray();
+            var min = array.Min();
+            Assert.Equal(min, 1);
+        }
```

Comment:
Yes, indeed, just wanted to get an initial feedback from you guys.
As for the plan, I am planning to write a generic code for testing that takes 1) an input 2) a lambda expression of how to verify the correctness of the code, and then I will pass multiple input types. This way I will avoid to write the code multiple times, yet I can ensure I cover different inputs with different types. Does that make sense?